.ci.yaml

```
@@ -214,6 +214,40 @@ targets:
       channel: stable
       version_file: flutter_stable.version
 
+  - name: Linux_android custom_package_tests master
+    bringup: true # New target
+    recipe: packages/packages
+    timeout: 30
+    properties:
+      add_recipes_cq: "true"
+      version_file: flutter_master.version
+      target_file: linux_custom_package_tests.yaml
+      cores: "32"
+      # Pigeon tests need Andoid deps (thus the Linux_android base) and
```

```suggestion
      # Pigeon tests need Android deps (thus the Linux_android base) and
```

Is this sufficient to name the suite "Linux_android" though? :P

---

I like andoid better

---

`Linux_andoid custom_package_tests master` :P